runtime/js/40_testing.js

```diff
@@ -34,6 +34,7 @@
     TypeError,
   } = window.__bootstrap.primordials;
 
+  let isOpCallTracingEnabled = false;
```

Comment:
Maybe we should ask the other module for this state so we have a single source of truth?

Reply:
That was my initial thought - but there might be some setup code execute before we call `Deno[Deno.internal].runTests()` and thus if we don't do it this way then we won't be able to collect traces for the setup code.

Reply:
Nvm, I see what you mean. I switched to using a function in `Deno.core` instead.